### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by afcj8** [badges/shields#10589] [Q&A] [answered]
When will Visual Studio Code icons be incorporated?

**Comment by jNullj** [comment]
As detailed in [this blog post](https://shields.io/blog/sunsetting-shields-custom-logos), icons are derived from the Simple Icons project. If you require a specific icon, you may contribute to the Simple Icons project or use a custom icon encoded in Base64 as a parameter.

**Reply by afcj8** [reply]
How do I contribute and use Base64?

**Reply by jNullj** [reply, marked as answer]
To use a base64 image:
1. Convert your custom logo image to Base64. You can use an online converter to do this. recommended using SVG
    Example Base64 string - `data:image/png;base64,iVBORw0KGg... `
2. Use the Shields.io badge URL format to include your Base64-encoded logo. `https://img.shields.io/badge/custom-badge-green?logo=data:image/png;base64,iVBORw0KGg...`
3.  you can see a full example [in the docs](https://shields.io/docs/logos#custom-logos)

To contribute new logos to the simple icons project i advice you to read how info [on their repository](https://github.com/simple-icons/simple-icons), Please note it might take some time for the added icon to be added to shields.io after a new icon is merged.
